docs/UserGuide.md

```diff
@@ -202,12 +235,16 @@ _Details coming soon ..._
 
 ## Command summary
 
-| Action     | Format, Examples                                                                                                                                                      |
-|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague` |
-| **Clear**  | `clear`                                                                                                                                                               |
-| **Delete** | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                   |
-| **Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                           |
-| **Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                            |
-| **List**   | `list`                                                                                                                                                                |
-| **Help**   | `help`                                                                                                                                                                |
+| Action         | Format, Examples                                                                                                                                       |
+|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**        | `add n/<name> p/<phone number> e/<email address> a/<address>` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665` |
+| **Delete**     | `delete <id>`<br> e.g., `delete 3`                                                                                                                     |
+| **Clear**      | `clear`                                                                                                                                                |
+| **Tag**        | `tag <id> l <tag name>`  <br> e.g., `tag 2 l friends`                                                                                                  |
+| **Delete Tag** | `delete-tag <id> l <tag name>` <br> e.g., `delete-tag 2 l friends`                                                                                     |
```

Comment:
delete-tag was renamed to `untag` and the format for tag and untag is `<id> t/<tag> [t/tag]...`